sdk/nodejs/Makefile

```diff
@@ -95,9 +95,11 @@ ifneq ($(PULUMI_TEST_COVERAGE_PATH),)
 	if [ -e .nyc_output ]; then yarn run nyc report -r cobertura --report-dir $(PULUMI_TEST_COVERAGE_PATH); fi
 endif
 	@cd cmd/pulumi-language-nodejs && $(GO_TEST_FAST) $(shell go list ./... | grep -v /vendor/ | grep -v templates)
+	$(GO_TEST_FAST) $(shell go list ./...)
```

Comment:
I don't think we were running the tests in `sdk/nodejs/npm` before, they all pass though 😅

Reply:
I think they ran with the go tests, because it's all the go sdk really